docs/detections/alerts-view-details.asciidoc

```diff
@@ -52,7 +52,7 @@ IMPORTANT: If you've enabled grouping on the Alerts page, the alert details flyo
 [[preview-panel]]
 === Preview panel 
 
-The preview panel shows more information about the rule associated with the alert and the reason the alert was generated. This panel overlays the right panel when you click **Show rule details** or **Show full reason** in the <<about-section,About>> section. To close the preview panel, click **x**. 
+Some areas in the flyout provide previews when you click on them. For example, clicking **Show rule summary** in the Rule description displays a preview of the rule's details. To close the preview, click **x**. 
```

Comment:
$0.02: I know the UI area is labelled "Rule description," but it doesn't seem so specialized that it needs to be capitalized in general references like this:
```suggestion
Some areas in the flyout provide previews when you click on them. For example, clicking **Show rule summary** in the rule description displays a preview of the rule's details. To close the preview, click **x**. 
```